openpype/hosts/nuke/plugins/load/load_clip.py

```diff
@@ -22,6 +22,8 @@
 )
 from openpype.hosts.nuke.api import plugin
 
+log = Logger.get_logger(__name__)
```

Comment:
I would not change calling of `self.log` in the plugin. If you need to override logger please do it on class level rather.